# Pose generator params

# sampling params

# step: defined in radians, this is the sampling resolution
# in other words a sample will be created with increments of this size
roll_step:     0.6
pitch_step:    0.4
yaw_step:      0.4

# the angular range in radians to begin, stop sampling
roll_start:   -3.14
roll_end:      3.14
pitch_start:  -0.4
pitch_end:     0.4
yaw_start:    -0.4
yaw_end:       0.4
